python/gubernator/gubernator_pb2.py

```diff
@@ -15,552 +14,41 @@
 from google.api import annotations_pb2 as google_dot_api_dot_annotations__pb2
```

Comment:
I'd note that this import doesn't appear to be used anymore, not sure what's happening there?